recipes/iptkl/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "iptkl" %}
+{% set version = "0.6.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 340dc6265e64a3341f0db05151edf251e1e6c8476401d2a10194fbbaeb16a7db
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - biopython
+    - bokeh
+    - colour
+    - h5py
+    - holoviews
+    - logomaker
+    - lxml
+    - matplotlib
+    - mhcnames
+    - nglview
+    - numba
+    - numba ==0.53.1
+    - pandas
+    - pip
+    - plotly
+    - pyopenms
+    - pyteomics
+    - python
+    - scipy
+    - seaborn
+    - sklearn
+    - statannot
+    - tqdm
+  run:
+    - biopython
+    - bokeh
+    - colour
+    - h5py
+    - holoviews
+    - logomaker
+    - lxml
+    - matplotlib
+    - mhcnames
+    - nglview
+    - numba
+    - numba ==0.53.1
+    - pandas
+    - plotly
+    - pyopenms
+    - pyteomics
+    - python
+    - scipy
+    - seaborn
+    - sklearn
+    - statannot
+    - tqdm
+
+test:
+  imports:
+    - IPTK
+    - IPTK.Analysis
+    - IPTK.Classes
+    - IPTK.IO
+    - IPTK.IO.Interfaces
+    - IPTK.Utils
+    - IPTK.Visualization
+
+about:
+  home: "https://github.com/ikmb/iptoolkit"
+  license: MIT
+  license_family: MIT
+  license_file: 
```

Comment:
please add LICENCE.txt here, as this is your license_file

Reply:
cool, I have added the licence files